packages/fxa-settings/src/components/ButtonDownloadRecoveryKeyPDF/requiredFont.tsx

```diff
@@ -156,6 +161,7 @@ export const getRequiredFont = (language: string) => {
           },
         ],
         direction: 'ltr',
+        breakwords: true,
```

Comment:
```suggestion
        breakwords: false,
```

Korean shouldn't need this since they do have spaces between words.

Reply:
Ah yes, thanks! Just added this change.